test/server/server_test.cc

```diff
@@ -612,7 +614,6 @@ TEST_P(ServerInstanceImplTest, LifecycleNotifications) {
     server_->run();
     handle1 = nullptr;
     handle2 = nullptr;
```

Comment:
nit: leave behind a comment where setting handle3=nullptr used to be indicating that it is now cleared in its own callback to test that flow.

Reply:
Done